client/components/Dummy.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import DummyChild from './DummyChild';
+export default class Dummy extends React.Component {
```

Comment:
nit: Add a new line above this line, I think it's generally better to divide the imports and the code